lib/fizzbuzz.rb

```diff
@@ -2,7 +2,21 @@ class SuperFizzBuzz
 
   def run(input)
 
-    #Implement your code here
+    result = ""
+
+    if(input % 3 == 0)
```

Comment:
looks good. these also can be wrote as oneliners. take a look at the `.tap` method it could come in handy here if you wanted to make this more concise. you'll see it often in some production code - super helpful when building up an object and then returning it.

```ruby
"".tap do |str|
  str << "Fizz"
end